CONTRIBUTORS.yaml

```diff
@@ -1873,6 +1873,10 @@ TKlingstrom:
     joined: 2018-11
     elixir_node: se
 
+tvedtees:
+    name: Eric Tvedte
+    joined: 2024-03
```

Comment:
Welcome! If you'd like to add any additional information (an orcid?) please feel free to!

Additionally if you want to add any funding sources that funded this work, please feel free to add those to `FUNDERS.yaml`